app/config/sources.inc.php

```diff
@@ -317,7 +322,7 @@
     ],
     'firefox/sendto.lang' => [
         'priority'          => 1,
-        'supported_locales' => $firefox_locales,
+        'supported_locales' => $mozillaorg,
```

Comment:
This is **NOT** part of this PR (never been). Leave the set of language as it is.